modules/miscutil/sql/tabfill.sql

```diff
@@ -571,6 +604,8 @@ INSERT INTO idxINDEX VALUES (25,'filetype','This index contains extensions of fi
 INSERT INTO idxINDEX VALUES (26,'miscellaneous','This index contains words/phrases from miscellaneous fields','0000-00-00 00:00:00', '', 'native','','No','No','No', 'BibIndexDefaultTokenizer');
 INSERT INTO idxINDEX VALUES (27,'doi','This index contains words/phrases from doi fields','0000-00-00 00:00:00', '', 'native','','No','No','No', 'BibIndexDOITokenizer');
 INSERT INTO idxINDEX VALUES (28,'filename','This index contains file names of files connected to records.','0000-00-00 00:00:00', '', 'native', '','No','No','No', 'BibIndexFilenameTokenizer');
+INSERT INTO idxINDEX VALUES (29,'country','This index contains country names of the affiliated institutes of the authors.','1970-01-01 00:00:00', '', 'native', '','No','No','No', 'BibIndexCountryTokenizer');
+INSERT INTO idxINDEX VALUES (30,'journalpage','This index contains words/phrases from the journal page field.','1970-01-01 00:00:00', '', 'native', '','No','No','No', 'BibIndexJournalPageTokenizer');
```

Comment:
While the "country" index sounds very useful, do you expect users to use "journalpage" index without corresponding "journaltitle" and "journalvolume" or "journalyear"?  That's why we had standardisation of everything into the unique "journal" index following standard form as defined by `CFG_JOURNAL_PUBINFO_STANDARD_FORM` conventions...